src/node_sqlite.cc

```diff
@@ -114,6 +116,19 @@ inline void THROW_ERR_SQLITE_ERROR(Isolate* isolate, const char* message) {
   }
 }
 
+inline void THROW_ERR_SQLITE_ERROR(Isolate* isolate, int errcode) {
+  const char* errstr = sqlite3_errstr(errcode);
+  Local<Object> e;
+
+  if (CreateSQLiteError(isolate, errstr).ToLocal(&e)) {
+    e->Set(isolate->GetCurrentContext(),
+           OneByteString(isolate, "errcode"),
```

Comment:
Adding an `errcode` similar to `CreateSQLiteError(Isolate* isolate, sqlite3* db)`. However, this seems to be an undocumented feature.